muesli/web/viewsLecture.py

```diff
@@ -84,9 +84,13 @@ def __init__(self, request):
     def __call__(self):
         lecture = self.db.query(models.Lecture).options(undefer('tutorials.student_count')).get(self.lecture_id)
         times = lecture.prepareTimePreferences(user=self.request.user)
-        subscribed = self.request.user.id in [s.id for s in lecture.students]
+        subscribed_tutorial = self.db.query(models.Tutorial).filter(
+            Tutorial.lecture_students.any(
+                LectureStudent.student_id == self.request.user.id and LectureStudent.lecture_id == self.lecture_id
+            )
+        ).first()
```

Comment:
Oh, that looks way better :) I will test this soon and incorporate it in my request when I have time.

Reply:
Hmm, this doesn't seem to work. First of all, I needed to replace `tutorials()` with `tutorials`, but even then,
```
self.request.user.tutorials.filter(Lecture == lecture).first()
```
returns `None` although the user is in a tutorial for the given lecture.

Reply:
I've managed to shorten the query to
```
self.request.user.tutorials.filter(LectureStudent.lecture_id == self.lecture_id).first()
```
But still, this doesn't appear ideal. Do you have any ideas on how to shorten it further?

Reply:
Oh, well, I think this is fine. The only other way I can think of is querying the tutorials and filtering lectures. Your message sounds like you would like to work on this further. I think this is as ideal, as it can get and am ready to merge it :)

I'll leave this open till you reply :D

Reply:
Okay, I've changed my opinion ;) I'm also good to go for now. If I change my opinion about this particular query sometime in the future, I'll just open a new PR.